src/views/Layout.css

```diff
@@ -60,3 +60,32 @@
 	text-decoration-thickness: 0.22em;
 	text-underline-offset: 0.1em;
 }
+
+/* List View Header Layout */
+.List {
+	display: flex;
+	flex-direction: column;
+	margin: 0 auto;
+	max-width: 100vw;
+	padding-top: calc(3rem + env(safe-area-inset-top, 1rem));
```

Comment:
Neat! This is very precise padding!

Reply:
Haha thank you, but I can't take credit! This was TCL's starter code, but I did play around with `position: absolute` when trying to see if I can get the oven card at the bottom center of the screen. That experiment didn't work out too well and I had to resort back to the original CSS format from `Layout.css`